portindex2json/portindex2json.tcl

```diff
@@ -64,7 +64,58 @@ proc remove_extra_braces {text} {
     return [regsub -all {[\{\}]} $text ""]
 }
 
-set fd [open [lindex $argv 0] r]
+proc print_usage {} {
+    puts stdout "\nUsage: /path/to/tclsh portindex2json.tcl <path-to-portindex> \[-info key=vlaue\]"
+    puts stdout "\nEach '-info key=value' combinaton adds one key-value pair to the JSON output inside the parent key: \"info\""
+    puts stdout "\nYou may provide any number of '-info key=value' combinations."
+    puts stdout "\nThe arguments <path-to-portindex> and `-info key=value` are position independent.\n"
```

Comment:
We could leave this information out, but if you keep it, keep the quotation marks consistent.